sql/src/test/java/io/crate/expression/scalar/DateFormatFunctionTest.java

```diff
@@ -123,8 +123,8 @@ public void testInvalidTimeZone() throws Exception {
 
     @Test
     public void testInvalidTimestamp() {
-        expectedException.expect(ConversionException.class);
-        expectedException.expectMessage("Cannot cast `'NO TIMESTAMP'` of type `text` to type `timestamp with time zone`");
+        expectedException.expect(IllegalArgumentException.class);
+        expectedException.expectMessage("Text 'NO TIMESTAMP' could not be parsed");
```

Comment:
This reverts changes made by https://github.com/crate/crate/pull/9763/files#diff-6723f5f075532e1729716bf3efe4a116R126